lib/src/utils.dart

```diff
@@ -2,7 +2,12 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'package:charcode/charcode.dart';
```

Comment:
Import the local `charcode.dart` instead.

Reply:
Done.